Mathlib/Data/NNRat/Defs.lean

```diff
@@ -367,6 +367,7 @@ theorem natAbs_num_coe : (q : ℚ).num.natAbs = q.num := rfl
 
 lemma coprime_num_den (q : ℚ≥0) : q.num.Coprime q.den := by simpa [num, den] using Rat.reduced _
 
+-- TODO: Rename `Rat.coe_nat_num`, `Rat.intCast_den`, `Rat.ofNat_num`, `Rat.ofNat_den` to match
```

Comment:
Can you take the updated version I gave instead? `Rat.coe_nat_num` has already been renamed.